test/notifications/notification_server_test.exs

```diff
@@ -1,13 +1,17 @@
 defmodule Notifications.NotificationServerTest do
-  use ExUnit.Case, async: false
+  use ExUnit.Case, async: true
```

Comment:
Is this safe to do with tests that access the database?

Reply:
probably not...checked this in by mistake.

Reply:
scratch that, I apparently did that for a reason. The reason is that, with `async: false`, I get the following quite verbose error:

```
     ** (DBConnection.OwnershipError) cannot find ownership process for #PID<0.8354.0>.

     When using ownership, you must manage connections in one
     of the four ways:

     * By explicitly checking out a connection
     * By explicitly allowing a spawned process
     * By running the pool in shared mode
     * By using :caller option with allowed process

     The first two options require every new process to explicitly
     check a connection out or be allowed by calling checkout or
     allow respectively.

     The third option requires a {:shared, pid} mode to be set.
     If using shared mode in tests, make sure your tests are not
     async.

     The fourth option requires [caller: pid] to be used when
     checking out a connection from the pool. The caller process
     should already be allowed on a connection.

     If you are reading this error, it means you have not done one
     of the steps above or that the owner process has crashed.

     See Ecto.Adapters.SQL.Sandbox docs for more information.
```

So I will investigate further what this means and how to resolve it.

Reply:
Looks like I just had to use `Skate.DataCase` here instead of `ExUnit.Case`, and a few other corresponding changes.